__init__.py

```diff
@@ -12,13 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from mycroft import MycroftSkill, intent_file_handler
+from mycroft import MycroftSkill, intent_file_handler, intent_handler
+from adapt.intent import IntentBuilder
+from datetime import datetime
 import sys
 import imaplib
 import email
 import email.header
+import email.utils
 
-def list_new_email(account, folder, password, port, address):
+EMAIL_POLL_INTERVAL = 60 # in seconds
```

Comment:
This seems like such a short time: maybe 2 min (120 sec)
```suggestion
EMAIL_POLL_INTERVAL = 120 # in seconds
```